deploy/storage/dev/local-storage/pv-setup/10_daemonset.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: pv-setup
+  labels:
+    app: pv-setup
+spec:
+  selector:
+    matchLabels:
+      app: pv-setup
+  template:
+    metadata:
+      labels:
+        app: pv-setup
+    spec:
+      hostPID: true
+      hostIPC: true
+      hostNetwork: true
+      serviceAccountName: pv-setup
+      containers:
+      - name: pv-setup
+        image: registry.access.redhat.com/ubi8/ubi:latest
+        imagePullPolicy: Always
+        command:
+        - "chroot"
```

Comment:
chroot means all the binaries you are using are from the host instead of the image where you can be sure they always are present and no OS change will break you

Reply:
Changed image to `quay.io/scylladb/scylla-operator-images:node-setup` (https://github.com/scylladb/scylla-operator-images/pull/1) and removed `chroot`.